dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/json/JsonParser.java

```diff
@@ -298,25 +294,7 @@ public DatasetVersion parseDatasetVersion(JsonObject obj, DatasetVersion dsv) th
             dsv.setArchiveTime(parseTime(obj.getString("archiveTime", null)));
             dsv.setUNF(obj.getString("UNF", null));
             // Terms of Use related fields
-            TermsOfUseAndAccess terms = new TermsOfUseAndAccess();
-            terms.setTermsOfUse(obj.getString("termsOfUse", null));
-            terms.setTermsOfAccess(obj.getString("termsOfAccess", null));
-            terms.setConfidentialityDeclaration(obj.getString("confidentialityDeclaration", null));
-            terms.setSpecialPermissions(obj.getString("specialPermissions", null));
-            terms.setRestrictions(obj.getString("restrictions", null));
-            terms.setCitationRequirements(obj.getString("citationRequirements", null));
-            terms.setDepositorRequirements(obj.getString("depositorRequirements", null));
-            terms.setConditions(obj.getString("conditions", null));
-            terms.setDisclaimer(obj.getString("disclaimer", null));
-            terms.setDataAccessPlace(obj.getString("dataAccessPlace", null));
-            terms.setOriginalArchive(obj.getString("originalArchive", null));
-            terms.setAvailabilityStatus(obj.getString("availabilityStatus", null));
-            terms.setContactForAccess(obj.getString("contactForAccess", null));
-            terms.setSizeOfCollection(obj.getString("sizeOfCollection", null));
-            terms.setStudyCompletion(obj.getString("studyCompletion", null));
-            terms.setLicense(parseLicense(obj.getString("license", null)));
```

Comment:
Ech. It seems that we still relying on `license`, `termsOfUse` and 'restrictions`. in DublinCore exporter and oaiore exporter (failed tests). We should correct tests and make 2 tasks that will fix those issues (exposing terms of use information). The failing test for native json exporter should be fixed as well

Reply:
Hmm, should I fixe tests or just revert the changes for JsonParser? Maybe getting rid of TermsOfUseAndAccess should just be seperate task?

Reply:
Maybe let's just revert the changes for JsonParser.